lib/resources/r4/charge_item_create.yaml

```diff
@@ -289,9 +289,9 @@ fields:
       }
     note: >
       <ul>
-        <li>Custom-attribute-name and Custom-attribute-value are both required</li>
+        <li>custom-attribute-name and custom-attribute-value are both required</li>
         <li>id and valueString for custom-attribute-value are required</li>
```

Comment:
should this be `custom-attribute-name` instead of `custom-attribute-value` ?

Reply:
updated - https://github.com/cerner/fhir.cerner.com/pull/799/commits/4c6beba5ca17e0ce2faa98beae7a5812099b613e